lib/sensors/led_driver.cpp

```diff
@@ -0,0 +1,97 @@
+#include "led_driver.hpp"
+
+namespace hyped::sensors {
+
+std::optional<LedDriver> LedDriver::create(core::ILogger &logger,
+                                           std::shared_ptr<io::II2c> i2c,
+                                           const std::uint8_t device_address)
+{
+  if (device_address != kDefaultLedDriverAddress) {
```

Comment:
what's the point of taking an argument for the LED driver if it must match the default?